cfpb/design-system
# Styling: Hover/Focus on TextInputs utilize dark colors

closes #1881 

## Changes
- Added new dark hover/focus colors to variables.md
- Added new dark hover/focus colors to text-inputs.md
- Updated text-inputs.md with the new dark hover/focus color stories

## How to Test 
- Test Link: https://deploy-preview-1886--cfpb-design-system.netlify.app/design-system/components/text-inputs
- **Code inspect** `text-inputs.less` and ensure no unintended styling changes affect other components
- Navigate to the `Variables` story section to view the new styling variables.
- **Navigate to the `TextInputs` story section and view the new dark hover/focus styles**. Hover and focus in the three normal status sections (success/error/warning)

## Screenshot(s)

Subtle dark colors -- only visible during hover (not focus)

|Before|After|
|---|---|
|<img width="204" alt="Screenshot 2024-01-22 at 3 43 24 AM" src="https://github.com/cfpb/design-system/assets/13324863/ed1d4083-8ac9-4827-991c-1a87d04c46df">|<img width="205" alt="Screenshot 2024-01-22 at 4 15 28 AM" src="https://github.com/cfpb/design-system/assets/13324863/bab0aacb-9cda-4fce-84f7-f2d888efeca6">|
